Update metadata R packaging with new workshop structure (#23)
esciencecenter-digital-skills/workshop-metadata: ds-rpackaging/schedule.md

This workshop is scheduled on four separate days. This allows you to work on

> your package in between, implementing the lessons you learned during our

Reviewer: ```suggestion
your package in between the session, and implementing the lessons you learned during our
```

> meetings together. Please schedule time in your calendar to do so, so you can

Reviewer: ```suggestion
meetings together. Please schedule dedicated work time in your calendar, so you can
```
